tests/RetryProxyTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Tests;
+
+use Keboola\DbExtractor\Logger;
+use Keboola\DbExtractor\Retry\RetryProxy;
+use Keboola\DbExtractor\Test\DataLoader;
+use Keboola\DbExtractor\Test\ExtractorTest;
+use Monolog\Handler\TestHandler;
+
+class RetryProxyTest extends ExtractorTest
```

Comment:
to add the codes into the extractor, we can have a `getSqlStateRetryCodes` method in the base class which would be used when creating instance of RetryProxy. that would be
implemented as `return []` (i can't imagine and default codes now, and would probably leave this until we agree in all extractors, that some code is repeated) - specific extractors can then override this methods

Reply:
Good idea.  I'll do this, thanks!